build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
     ext.app_center_sdk_version = '1.11.0'
     ext.stripe_sdk_version = '10.4.6'
     ext.material_library_version = '1.0.0'
-    ext.traveler_core_version = '0.20.1'
+    ext.traveler_core_version = '0.21.0'
```

Comment:
why are you downgrading the version here?

Reply:
@OmarPadierna upgrading -> `0.21.0 > 0.20.1`

Reply:
whenever you changed core you push your changes and then when the changes are merged to master a new core build is ready to use. then you ping me and i push a new ui sdk with updated core version or you can make a branch change the ui core version to latest and send a pr.

Reply:
Ok, I removed this update. Only fix for encoding is in PR now